docs/en/ingest-management/elastic-agent/uninstall-elastic-agent.asciidoc

```diff
@@ -22,17 +22,39 @@ stops and uninstalls any managed programs, such as {beats} and
 
 If you run into problems, see <<fleet-troubleshooting>>.
 
+For DEB or RPM installations, you must uninstall the {agent} manually.
+
+//QUESTION: Is this true ^^? Are there any other ways to uninstall that are less
+//manual?
+
 [discrete]
-== Uninstall on DEB or RPM
+== Uninstall manually
```

Comment:
```suggestion
== Step-by-step uninstall
```

I would prefer some other word than 'manual' here because it is kind of vague, and typing in 'elastic-agent uninstall' is a manual process anyways, so differentiating them is desirable.  if you don't like 'step-by-step' that is ok by me, happy to hear other suggestions.  and you can merge as is, too, without changes if you want to get it in and iterate / discuss later

Reply:
I'll change it to "Manually remove {agent} files" I wouldn't describe running the `install` command as a manual process because it sets a lot of stuff up for you. Since `uninstall` is the opposite of `install`, I wouldn't want to describe the uninstall command as manual either.